GsaGH/Parameters/5_Results/MeshResultGoo.cs

```diff
@@ -1,25 +1,27 @@
-﻿using Grasshopper.Kernel;
+﻿using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using Grasshopper.Kernel;
 using Grasshopper.Kernel.Types;
 using OasysUnits;
 using Rhino.Geometry;
-using System.Collections.Generic;
-using System.Linq;
 
 namespace GsaGH.Parameters
 {
   public class MeshResultGoo : GH_GeometricGoo<Mesh>, IGH_PreviewData
   {
-    internal List<List<IQuantity>> ResultValues;
-    internal List<List<Point3d>> Vertices;
-
+    public List<List<IQuantity>> ResultValues = new List<List<IQuantity>>();
```

Comment:
if we don;t want to change it outside - make it readonly

Reply:
users might want to access this via code from grasshopper (c# / python)